src/components/Shared/DocHeader.tsx

```diff
@@ -0,0 +1,77 @@
+import { cx } from 'class-variance-authority'
+import { H1 } from '@/components/MDX/H1'
+import { P } from '@/components/MDX/P'
+import { Icon } from '@spark-ui/icon'
+import { Tag } from '@spark-ui/tag'
+
+import { Npm } from '@/icons/Npm'
+import { GitHub } from '@/icons/GitHub'
+import { Bug } from '@/icons/Bug'
+import { WarningFill } from '@spark-ui/icons'
+
+export const DocHeader = ({
+  title,
+  name,
+  category,
+  license,
+  description,
+  keywords,
+  packageUrl,
+  className,
+  version,
+  bugReportUrl,
+}) => {
+  return (
+    <header className={cx('bg-background-variant', className)}>
```

Comment:
The page already has a header 🤔

Reply:
a header inside an article is different
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header#accessibility